io/plugins/eu.esdihumboldt.hale.io.xls/plugin.xml

```diff
@@ -76,8 +76,8 @@
                ref="eu.esdihumboldt.hale.io.xls.xlsx">
          </contentType>
          <providerParameter
-               description="States if the first line in the CSV file should be skipped (e.g. because it contains the header)"
-               label="Skip first line"
+               description="States how many lines should be skipped when reading instances from a XLS file (e.g. because of the header)"
+               label="Skip N lines"
```

Comment:
The name is correct here :+1: 

Over than that same things as for CSV apply:
- mention backwards compatible behavior?
- parameterBinding/valueDescriptor further below were not updated